File: Assignment9/payCheck.c
/*
Program: 	A C program to compute employees paycheck
Author: 	Ayodele Olufemi
Date: 		11/03/2021
*/

#include<stdio.h>

int main() {

	// Creating variable for number of employees
	int numEmployees;

	// Requesting for number of employees
	printf("Enter the number of employees: ");
	scanf("%d", &numEmployees);

	// Creating needed variables for use in loop
	int i = 1;
	float hourlyRate;
	float hoursWorked;
	float grossPay;
	float tax;
	float netPay;

	
	// Creating a loop to compute employee's paycheck
	while (i <= numEmployees) {
		// Printing header
		printf("------------------------Employee #%d------------------------\n", i);

		// Requesting for employee's hourly rate
		printf("Enter the hourly rate: ");
		scanf("%f", &hourlyRate);

		// Requesting for employee's hours worked
		printf("Enter the hours worked: ");
		scanf("%f", &hoursWorked);
		
		// Calculating gross pay
		grossPay = hourlyRate * hoursWorked;

		// Calculating tax
		if (grossPay <= 500) {
			tax = 0;
		} else if (grossPay <= 1100) {
			tax = grossPay * 0.02;
		} else if (grossPay <= 12000) {
			tax = grossPay * 0.05;
		} else {
			tax = grossPay * 0.07;
		}

		// Calculating net pay
		netPay = grossPay - tax;

		// Printing the results
		printf("\nHourly Rate: $%.2f\nHours: %.2f hours\nPay: $%.2f\nTax: $%.2f\nNet Pay: $%.2f\n", hourlyRate, hoursWorked, grossPay, tax, netPay);
		
		// incrementing processed employee counter
		i++;
	}

	return 0;
}

File: Assignment9/payCheck.py
'''
Program: 	A Python program to compute employees paycheck
Author: 	Ayodele Olufemi
Date: 		11/03/2021
'''

# Requesting for number of employees
numEmployees = int(input("Enter the number of employees: "))

# Creating a loop to compute employee's paycheck
for i in range(1, numEmployees + 1):
    # Printing header
    print(f"------------------------Employee #{i}------------------------\n")

    # Requesting for employee's hourly rate
    hourlyRate = float(input("Enter the hourly rate: "))

    # Requesting for employee's hourly worked
    hoursWorked = float(input("Enter the hours worked: "))

    # Calculating gross pay
    grossPay = hourlyRate * hoursWorked

    # Calculating tax
    if (grossPay <= 500):
        tax = 0
    elif (grossPay <= 1100):
        tax = grossPay * 0.02
    elif (grossPay <= 12000):
        tax = grossPay * 0.05
    else:
        tax = grossPay * 0.07

    # Calculating net pay
    netPay = grossPay - tax

    # Printing the results
    print(f"Hourly Rate: ${hourlyRate:.2f}\nHours: {hoursWorked:.2f} hours\nPay: ${grossPay:.2f}\nTax: ${tax:.2f}\nNet Pay: ${netPay:.2f}\n")
